visualization/app/codeCharta/ui/codeMap/rendering/codeMapBuilding.ts

```diff
@@ -113,4 +110,8 @@ export class CodeMapBuilding {
 		this._defaultDeltaColor = color
 		this._deltaColor = color
 	}
+
+	setOnclickDeltaColor(color: string) {
```

Comment:
This and the above function should probably be renamed to stay consistent with `setColor`